### PR TITLE
main/bash-completion: update to 2.13.0

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

File: main/bash-completion/template.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
...
 url = "https://github.com/scop/bash-completion"
 source = f"{url}/releases/download/{pkgver}/{pkgname}-{pkgver}.tar.xz"
-sha256 = "73a8894bad94dee83ab468fa09f628daffd567e8bef1a24277f1e9a0daf911ac"
+sha256 = "c5f99a39e40f0d154c03ff15438e87ece1f5ac666336a4459899e2ff4bedf3d1"
 # what's the point (needs a truckload of checkdepends too)
 options = ["!check"]
+
+
+def post_install(self):
+    self.rm(self.destdir / "etc/bash_completion.d", recursive=True)
```
Comment (nekopsykose) [inline main/bash-completion/template.py:22]: this technically places a compat function thing in there but idk if it's that important

Comment (triallax) [reply main/bash-completion/template.py:22]: you can pass `compatdir=/usr/share/bash-completion/completions` in `make_install_args` like i do in https://github.com/chimera-linux/cports/pull/1826 i guess

Comment (triallax) [reply main/bash-completion/template.py:22]: hmm actually then you'd need to either patch `bash_completion` to use that or set the `BASH_COMPLETION_COMPAT_DIR` envvar somewhere to use it, so i guess we can just remove the file if it's not important

Comment (nekopsykose) [reply main/bash-completion/template.py:22]: yeah will just leave it out for now i guess

Comment (nekopsykose) [reply main/bash-completion/template.py:22]: ah no, _get_cword is compat and needed by some things

Comment (triallax) [reply main/bash-completion/template.py:22]: bleh

**Pull request merged**